macos_bundle.sh

```diff
@@ -51,21 +49,24 @@ EOM
 
 cp build/src/gqrx Gqrx.app/Contents/MacOS
 cp resources/icons/gqrx.icns Gqrx.app/Contents/Resources
-cp /usr/local/lib/SoapySDR/modules*/libPlutoSDRSupport.so Gqrx.app/Contents/soapy-modules
-cp /usr/local/lib/SoapySDR/modules*/libremoteSupport.so Gqrx.app/Contents/soapy-modules
+# see https://apple.stackexchange.com/questions/437618/why-is-homebrew-installed-in-opt-homebrew-on-apple-silicon-macs
+MACDEPLOYQT6=/usr/local/opt/qt@6/bin/macdeployqt
+if [ -f /opt/homebrew/opt/qt@6/bin/macdeployqt ]; then
+    MACDEPLOYQT6=/opt/homebrew/opt/qt@6/bin/macdeployqt
+fi
+cp /*/*/lib/SoapySDR/modules*/libPlutoSDRSupport.so Gqrx.app/Contents/soapy-modules
+cp /*/*/lib/SoapySDR/modules*/libremoteSupport.so Gqrx.app/Contents/soapy-modules
```

Comment:
Instead of these changes, we could probably just replace the start of Homebrew paths with `$(brew --prefix)`.